.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+# Configuration file for pre-commit (https://pre-commit.com/).
+# Please run `pre-commit run --all-files` when adding or changing entries.
+
+repos:
+-   repo: https://github.com/koalaman/shellcheck-precommit
+    rev: 81c4cafc55d9f3da32b88aaad05ffb50a52ddfb2 # v0.8.0
```

Comment:
FYI, the idea is to use the commit ID rather than the version tag to ensure reproducibility.